src/github.com/stellar/horizon/actions_friendbot.go

```diff
@@ -18,7 +18,7 @@ func (action *FriendbotAction) JSON() {
 		action.CheckEnabled,
 		action.LoadAddress,
 		action.LoadResult,
-		action.LoadResource,
+		action.loadResource,
```

Comment:
This is an in-process refactoring/delinting.  Eventually every action will have it's helper methods made internal (in go, symbols with an initial capital letter are exported, lower case are private).

I'm simply performing the delinting as I have to touch files in the course of implementing the importer.  I'll probably do a big sweep when preparing this for PR and hit all the remaining files.

This particular refactoring:  I would like to minimize both "go lint" and "go vet" warnings.  Since initially writing this code, I have got the warnings built into my workflow so they are much easier to fix in the course of coding.

`FriendbotAction` itself is a little different, because it reuses `loadResource` from the `TransactionCreateAction` that it wraps.  If you notice, `FriendbotAction` itself doesn't even define a `LoadResource` or `loadResource` method.